.gitignore

```diff
@@ -64,6 +64,8 @@ obj
 *.vsix
 .vscode-test
 test-results.xml
+dist
```

Comment:
You should open up your vsix and make sure there's no unintended stuff in there

Reply:
Done